theme/sass/components/_card-component.scss

```diff
@@ -98,6 +98,8 @@
         &--horizontal {
           @include breakpoint($md) {
             flex-direction: row;
+            width: 60%;
+            float: left
           }
```

Comment:
Width and float left should be on the card component container not on the card which is where you've put it here.

Reply:
done